docs/HowToFAQ/FAQFunctionUnroll0.dfy

```diff
@@ -1,27 +1,27 @@
-function LengthSum(s: seq<string>): nat {
+function Sum(s: seq<string>): nat {
```

Comment:
I was suggesting to use a sequence of numbers

Reply:
Done.